packages/client/src/__tests__/migrations.test.ts

```diff
@@ -245,4 +245,59 @@ describe("Migrations", () => {
       ).rejects.toThrow(HTTPSErrors.Unauth);
     });
   });
+
+  describe("migrateCategoriesToArray", () => {
```

Comment:
Yeah...this is what I was talking about, `migrateCustomersToMultipleCategories` or `migrateCustomersToCategoriesArray` would make more sense. Or namespaced version (as you've suggested), something like this: `customersToPluralCategories`